### ISSUE TITLE
mermaid logo proposal

### ISSUE THREAD

**Issue by brunolnetto** [mermaid-js/mermaid#4005]
### Proposal

This seems a proper place to offer this image as repository proposal. See Screenshots

### Colors

RGB [0.933, 0.51, 0.933]

### Screenshots

https://imgur.com/a/XC0gZ3q

**Comment by brunolnetto** [comment]
@codehangen it is a very bold proposal: change the organization logo o something minimalistic. OR have minor logos to make it possible to use it as favicon.ico, for example.

**Comment by sidharthv96** [comment]
@brunolnetto, thank you for the proposal, the logo looks neat!

Unfortunately, we already made a logo change recently from the old one

![image](https://user-images.githubusercontent.com/10703445/217733404-8ba52760-7492-4429-9ef3-17a5a60f2d24.png)

To 

<img width="226" alt="image" src="https://user-images.githubusercontent.com/10703445/217733475-8ae0231a-aced-4b63-b3da-557d6dd9910f.png">

So we won't be accepting the proposal at this time. 

But I did notice that the logo is missing from our readme, so I'll be adding it there. Thank you!
